frontend/src/features/public-form/components/FormStartPage/useFormBannerLogo.tsx

```diff
@@ -1,35 +1,45 @@
 import { useMemo } from 'react'
 
-import { AdminFormDto, PublicFormDto } from '~shared/types'
-import { FormLogoState } from '~shared/types/form/form_logo'
+import { AgencyBase } from '~shared/types'
+import { FormLogo, FormLogoState } from '~shared/types/form/form_logo'
 
-import { useEnv } from '~features/env/queries'
-
-export const useFormBannerLogo = (form?: AdminFormDto | PublicFormDto) => {
-  const { data: { logoBucketUrl } = {} } = useEnv(
-    form?.startPage.logo.state === FormLogoState.Custom,
-  )
+interface UseFormBannerLogoInputs {
+  logoBucketUrl?: string
+  logo?: FormLogo
+  agency?: AgencyBase
```

Comment:
nit: use AgencyDto instead?

Reply:
the type of `form.admin.agency` is `AgencyBase` though, and none of the extra info is needed from `AgencyDto`. is there a reason for using `AgencyDto` instead?

Reply:
because that's the real field that is being used, but i guess that's fine too.